test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,157 @@
+      """
+
+  Scenario: User can create new namespace coolstore
+   Given Minishift has state "Running"
+    When executing "oc new-project coolstore" succeeds
+    Then stdout should contain
+     """
+     Now using project "coolstore"
+     """
+  
+  Scenario: User can create a template for coolstore 
+   Given Minishift has state "Running"
+    When executing "oc create -f https://raw.githubusercontent.com/jbossdemocentral/coolstore-microservice/1.0.x/openshift/coolstore-template.yaml" succeeds
+    Then stdout should contain
+     """
+     template "coolstore" created
+     """
+
+  Scenario: User can deploy the coolstore template to namespace coolstore 
+   Given Minishift has state "Running"
+    When executing "oc new-app --template=coolstore" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+
+  Scenario: Successfully built the Web UI service 
+   Given Minishift has state "Running" 
+    When executing "oc logs bc/web-ui --follow" succeeds
+    Then stdout should contain
+     """
+     Push successful
+     """
+
+  Scenario: Successfully built the inventory service 
+   Given Minishift has state "Running" 
+    When executing "oc logs bc/inventory --follow" succeeds
+    Then stdout should contain
+     """
+     Push successful
+     """
+
+  Scenario: Successfully built the catalog service 
+   Given Minishift has state "Running" 
+    When executing "oc logs bc/catalog --follow" succeeds
+    Then stdout should contain
+     """
+     Push successful
+     """
+
+  Scenario: Successfully built the cart service 
+   Given Minishift has state "Running" 
+    When executing "oc logs bc/cart --follow" succeeds
+    Then stdout should contain
+     """
+     Push successful
+     """
+
+  Scenario: Successfully deployed cart service to namespace coolstore 
+   Given Minishift has state "Running" 
+    When executing "oc rollout status deploymentconfig cart --watch" succeeds
+    Then stdout should contain
+     """
+     "cart" successfully rolled out
+     """
+
+  Scenario: Successfully built the gateway 
+   Given Minishift has state "Running" 
+    When executing "oc logs bc/coolstore-gw --follow" succeeds
+    Then stdout should contain
+     """
+     Push successful
+     """
+
+  Scenario: Successfully rolled out gateway to namespace coolstore 
+   Given Minishift has state "Running" 
+    When executing "oc rollout status deploymentconfig coolstore-gw --watch" succeeds
+    Then stdout should contain
+     """
+     "coolstore-gw" successfully rolled out
+     """
+
+  Scenario: Successfully deployed all services
+   Given Minishift has state "Running" 
+    When executing "oc status -v" succeeds
+    Then stdout should not contain
+     """
+     has failed
+     """
+
+  Scenario: Able to interact with the catalog from the coolstore gateway 
+   Given Minishift has state "Running" 
+    When set variable "apigw" executing "oc get pods -o name -l application=coolstore-gw"
```

Comment:
What would you suggest. I would see it similarly to a shell variable, which is therefore global for the execution of that shell instance and all calls inherit. So global... And allow to be overwritten when a new set in initiated. This is not accounted for at this moment as I append...